style.css

```diff
@@ -110,3 +141,228 @@ li:nth-child(2) {
   height: 400px;
   background-image: url(./images/Illustration.png);
 }
+
+/* works section */
+.works {
+  width: 100%;
+  margin-top: 100px;
+  display: grid;
+  row-gap: 50px;
+}
+
+.works .title {
+  font-family: 'Crete Round', serif;
+  font-weight: 400;
+  color: #1c1a19;
+  font-size: 30px;
+  line-height: 52px;
+  margin-top: 70px;
+}
+
+.title h2 {
+  margin-left: 30px;
+}
+
+.line-break {
+  height: 1px;
+  width: 90%;
+  margin: 30px auto 20px 10px;
+  background-color: #1c1a19;
+}
+
+.card-works {
+  padding: 20px;
+  display: grid;
+  row-gap: 50px;
+  align-items: flex-start;
+  overflow: hidden;
+}
+
+.card-work1 {
+  display: flex;
+  padding: 3px;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+.card-work {
+  width: 95%;
+  margin: auto;
+  height: 400px;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  background-image: url(./images/Img\ Placeholder\ 2.png);
+  background-size: cover;
+  overflow: hidden;
+}
+.card-work,.card-work1{
+  width: 100%;
+  margin: 40x;
+}
+
+.card-work1 img {
+  width: 100%;
+  margin-bottom: 20px;
+}
+
+.card-works h3 {
+  font-family: 'Crete Round', serif;
+  font-weight: 400;
+  font-size: 32px;
+  margin: auto auto 10px 12px;
+}
+
+.card-work1 h3 {
+  width: 327px;
+  height: 36px;
+  color: #091e42;
+}
+
+.card-work h3 {
+  width: 295px;
+  height: 81px;
+  color: #fff;
+}
+
+.card-works p {
+  width: 70%;
+  font-family: 'Inter', sans-serif;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  margin:0 0 10px 20px  ;
+}
+
+.card-work1 p {
+  color: #091e42;
+}
+
+.card-work p {
+  color: #fff;
+}
+
+.card-works ul {
+  width: 100%;
+  height: 50px;
+  margin-bottom: 20px;
+  display: flex;
+}
+
+.card-works button {
+  height: 48px;
+  background-color: #ff6b00;
+  font-weight: 700;
+  font-size: 17px;
+  border: #ff6b00;
+  color: #fff;
+}
+
+.card-work1 button {
+  width: 140px;
+  height: 55px;
+  margin: 10px auto auto 10px;
+}
+
+/* about me section */
+
+.about-me {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  background-color: #1c1a19;
+}
+
+.about-me .me-content {
+  height: 101vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding-left: 31px;
+  background-image: url(./images/Group.png);
+  background-repeat: no-repeat;
+  background-position: 100% 100%;
+  overflow: hidden;
+}
+
+
+.about-me .title h2 {
+  font-family: 'Crete Round', serif;
+  font-size: 45px;
+  line-height: 88px;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+}
+
+.about-me p {
+  width: 90%;
+  height: 30vh;
+  font-family: 'Inter', sans-serif;
+  font-size: 16px;
+  line-height: 24px;
+  color: #f4f5f7;
+}
+
+.about-me button {
+  width: 40%;
+  height: 55px;
+  background-color: #ff6b00;
+  font-family: 'Inter', sans-serif;
+  font-weight: bold;
+  font-size: 14px;
+  line-height: 24px;
+  color: #f4f5f7;
+  border: #ff6b00;
+}
+
+button:hover {
```

Comment:
- [ ] [OPTIONAL] This was just awesome adding hover effect on your buttons, the only property you have forgotten is the `cursor: pointer` because it is not currently available on your buttons. By setting up this property it will let the user understand that it's really a button because the cursor will change from an arrow to a hand. 😉